app/ui/legacy/src/main/res/layout/fragment_settings_list.xml

```diff
@@ -1,8 +1,31 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.recyclerview.widget.RecyclerView
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/settings_list"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
-    android:scrollbars="vertical" />
+    android:fitsSystemWindows="false"
+    android:gravity="center">
+
+    <RelativeLayout
```

Comment:
The fragment is loading fine. It's just the list that isn't displayed. That's because this `RelativeLayout` is missing a constraint for the top.